ycmd/completers/php/ci_completer.py

```diff
@@ -0,0 +1,117 @@
+#/usr/bin/env python
```

Comment:
This file needs to be reformatted to follow the ycmd style. Plenty of work to do here.